Aztec/Classes/TextKit/TextView.swift

```diff
@@ -242,12 +242,22 @@ open class TextView: UITextView {
             .font: defaultFont,
             .paragraphStyle: defaultParagraphStyle,
         ]
-        if let color = textColor {
+        if let color = defaultTextColor {
             attributes[.foregroundColor] = color
         }
         return attributes
     }
-    
+
+    open var defaultTextColor: UIColor?
+
+    override open var textColor: UIColor? {
+        get {
+            return super.textColor
+        }
+        set {
+            super.textColor = newValue            
+        }
+    }
```

Comment:
Do we need to override `textColor`? It seems to not do anything special 🤔

Reply:
Good catch, I was trying to catch when textColor was being changed and add those, let me remove them.

Reply:
Addressed by https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1237/commits/ba8524aba1332550efb05cad583a85ed3511beb5